config/default/manager_auth_proxy_patch.yaml

```diff
@@ -15,7 +15,7 @@ spec:
           capabilities:
             drop:
               - "ALL"
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
+        image: quay.io/brancz/kube-rbac-proxy:v0.18.2
```

Comment:
Instead of pinning to what looks like a personally hosted image (though I see that's what's suggested in the documentation) I do think we should remove this file entirely. :thinking:

Reply:
@shaneutt  should i remove this file ?

Reply:
Yes, let's remove it for now and if there end up being problems we can go "fix it forward" style. Thanks!

Reply:
deleted and updated pull request